video_xblock/video_xblock.py

```diff
@@ -385,12 +385,15 @@ def studio_view(self, _context):
             'players': PlayerName,
             'sources': TranscriptSource.to_dict().items(),
             # transcripts context:
-            'transcripts': transcripts,
+            'transcripts': filter_transcripts_by_source(
+                transcripts, sources=[TranscriptSource.THREE_PLAY_MEDIA], exclude=True
```

Comment:
This is because of the way @z4y4ts refactored transcripts studio view UI. Transcripts section is rendered via an accordion widget which has two mutually exclusive views: 3PlayMedia / default+manual transcripts. So we don't need 3PlayMedia transcripts in the context anymore.